namespaces/cloud-platform-live-0.k8s.integration.dsd.io/kube-ops/00-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-ops
```

Comment:
Missing labels & annotations. You might want to generate this and all other base namespace resources using the templates.